### PR TITLE
Fix newlines in response

### LINKED ISSUE #5521: Newlines Issue
**Rasa version**:
Rasa 1.8.2

**Python version**:
Python 3.6

**Operating system** 
Kubuntu

**Issue**:
When using the Response Selector as described in the rasa [Documentation](https://rasa.com/docs/rasa/user-guide/building-assistants/#build-faq-assistant), Newlines (```\n```) set in the responses.md File don't work and are just displayed as ```\n```. It seems the Backslash itself is getting escaped so that the String send to a Messanger is actually ```\\n```, which is then getting rendered as ```\n```. I only tested this with the Telegram Channel and with ```rasa shell``` but it might very well be a Problem when using other Channels as well. In addition the same Problem seems to appear when I don't put Quotation Marks around an Utter in the domain.yml.

### PR DESCRIPTION
**Pull request by m-vdb**
**Proposed changes**:
Fixes #5521
- Support new lines in `responses.md`

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: changelog/5521.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Responses used in ResponseSelector now support new lines.
```
Comment (dakshvar22) [inline changelog/5521.bugfix.rst:1]: As far as I understand, this would work if we explicitly use `\n` in the response text, right?

**Comment by dakshvar22** [comment]
Related to https://github.com/RasaHQ/rasa/issues/5800
@m-vdb The above issue mentions that adding `\n` explicitly already works?
Doesn't this fix also need you to add `\n` explicitly?

**Comment by m-vdb** [comment]
@dakshvar22 it's weird I cannot reproduce the behaviour that the user described in #5800 🤔 When someone adds `\n` in `data/responses.md`, the newline character is de facto escaped. The only way to have it unescaped, is to press ENTER, but this would break the current format. So at the moment there is no way to support newlines other than writing `\n`.

Is it possible that the user is describing that it works because the channel he uses automatically unescapes newlines? (neither `rasa shell` or Rasa X do this)

**Comment by m-vdb** [comment]
build blocked by https://github.com/RasaHQ/rasa/pull/5879

File: changelog/5521.bugfix.rst (hunk 1 above)
Comment (dakshvar22) [inline changelog/5521.bugfix.rst:1]: Just to be explicit.
```suggestion
Responses used in ResponseSelector now support new lines with explicitly adding ``\n`` between them.
```

**Comment by dakshvar22** [comment]
@m-vdb Yeah, I just tested on my side too. Atleast Rasa shell wouldn't do it.

**Comment by m-vdb** [comment]
👍 thanks for the changelog suggestion, way better

**Comment by code-jam20** [comment]
Hi ,

Can you please tell which version of rasa has this fixed available?
I am facing similar problems with the Facebook messenger channel.

Thanks!

**Comment by code-jam20** [comment]
@m-vdb @dakshvar22 I see that last release happened on May 15 only, could you please release this fix?

**Comment by m-vdb** [comment]
sure @code-jam20 we'll try to push a micro release in the coming days 👍

**Comment by m-vdb** [comment]
@code-jam20 this fix is now released as part of [`1.10.2`](https://pypi.org/project/rasa/)

**Pull request merged**